dcplib/trufflehog_security_regex/README.md

```diff
@@ -0,0 +1,19 @@
+Trufflehog Regex Patterns
+=========================
+
+`trufflehog_regex_patterns.json` is a set of custom regex patterns for use with trufflehog.
+
+In a CI/CD pipeline, one can use the following script:
+
+    wget -O regex.json https://raw.githubusercontent.com/HumanCellAtlas/dcplib/tree/master/dcplib/trufflehog_security_regex/trufflehog_regex_patterns.json
+    trufflehog --regex --rules regex.json --entropy=False https://github.com/HumanCellAtlas/data-store.git
+    rm regex.json
+
+This will check the entire git history of a repository for the patterns inside of `trufflehog_regex_patterns.json`, 
+where in this case the example repository is: https://github.com/HumanCellAtlas/data-store.git
+
+Notes:
+1. `--entropy=False` turns off high entropy string matching, otherwise any uuid seems to trigger it.
```

Comment:
This flag seems unfortunate since the things we'd like to detect, strong passwords and keys, have high entropy.

Is there an alternate strategy?

Reply:
trufflehog itself is a fairly small script.  We could write our own function and inherit most of trufflehog around it.  Or just run our own script.  He has links to the logic behind the entropy algorithm used, and I could try to find something more suitable for our needs.

I don't think we can use it as is though, it generates gobs of pages though when run.

Reply:
Looking into this a bit more... I looked at making a whitelist for uuids, or extending the length of the hash required but since this is intended for many repos, the whitelist would need to be custom for each.  We should talk about this more, but I think maybe that would be better as a separate PR, and maybe it's worth consulting DevSecOps as to their requirements.